src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,53 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_STORAGE_SIZE = 9;
+    private K[] keys;
+    private V[] values;
+    private int currentStorageSize = 0;
```

Comment:
we don't need to initialize `size` variable explicitly
```suggestion
    private int size;
```